autotest/utilities/test_ogr2ogr_lib.py

```diff
@@ -490,9 +491,14 @@ def test_ogr2ogr_clipsrc_no_dst_geom():
 
     tmpfilename = "/vsimem/out.csv"
     wkt = "POLYGON ((479461 4764494,479461 4764196,480012 4764196,480012 4764494,479461 4764494))"
-    ds = gdal.VectorTranslate(
-        tmpfilename, "../ogr/data/poly.shp", options='-f CSV -clipsrc "%s"' % wkt
-    )
+    if args_using_options:
+        ds = gdal.VectorTranslate(
+            tmpfilename, "../ogr/data/poly.shp", options='-f CSV -clipsrc "%s"' % wkt
+        )
+    else:
+        ds = gdal.VectorTranslate(
```

Comment:
To make things simpler, I would just test this branch.  The use of -clipsrc was only because there was no corresponding keyword option